data/json/items/melee/unarmed_weapons.json

```
@@ -119,5 +119,62 @@
     "flags": [ "STAB", "UNARMED_WEAPON", "DURABLE_MELEE", "SHEATH_KNIFE" ],
     "price": 10000,
     "qualities": [ [ "CUT", 1 ], [ "BUTCHER", 14 ] ]
+  },
+  {
+    "id": "katar",
+    "type": "GENERIC",
+    "category": "weapons",
+    "weapon_category": [ "FIST_WEAPONS" ],
+    "symbol": "!",
+    "color": "dark_gray",
+    "name": { "str": "katar" },
+    "looks_like": "knuckle_katar",
+    "description": "A steel brace with a metal blade at the end, designed to allow the user to stab and slash with the same motions as a punch.",
+    "material": [ "steel" ],
+    "volume": "750 ml",
+    "cutting": 20,
+    "price": "500 USD",
+    "price_postapoc": "5 USD",
```

```suggestion
    "name": { "str": "katar" },
    "copy-from": "knuckle_katar",
    "description": "A steel brace with a metal blade at the end, designed to allow the user to stab and slash with the same motions as a punch.",
    "material": [ "steel" ],
    "volume": "750 ml",
    "cutting": 20,
    "price": "500 USD",
```
Just realized this doesn't have a weight attached, setting it to copy from the razor katar will alllow re-using the symbol, color, post-apoc price, and weight along with gaining its baseline bash damage (likely for the best since its damage over the punch dagger's total is only 4 points) and blocking technique.